install/runtime-payload/hack/build-image.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+script_dir=$(dirname "$(readlink -f "$0")")
+
+registry="${registry:-quay.io/confidential-containers/peer-pods-runtime-payload}"
+TARGET_ARCH="${TARGET_ARCH:-amd64}"
+IMAGE_TAG="${IMAGE_TAG:-dev}"
+
+function setup_env_for_arch() {
+	case "${TARGET_ARCH}" in
+		"amd64")
+			image="quay.io/confidential-containers/runtime-payload:kata-containers-b11b6e3756a15e602a5bff1d4d47da8ecf23c593"
+			kernel_arch="x86_64"
+			;;
+		"s390x")
+			image="quay.io/confidential-containers/runtime-payload:kata-containers-b11b6e3756a15e602a5bff1d4d47da8ecf23c593"
+			# TODO change to "quay.io/confidential-containers/runtime-payload:kata-containers-latest" when s390x image is available.
+			kernel_arch="s390x"
+			;;
+		(*) echo "$1 is not supported" && exit 1
+	esac
+}
```

Comment:
Hey DaLi, this change has made me realise that we are pulling in a ~4 month old version of the kata-runtime payload on x86. I'm not sure what the best strategy is, but maybe updating it to use quay.io/confidential-containers/runtime-payload-ci:kata-containers-latest as standard, which is the latest version of CCv0 (and mutlti-arch) would be better? then we can pick up the specific runtime-payload:commit versions that CoCo releases as part of our peer pods release process (until we find a way to merge the payload images entirely) @bpradipt @jensfr - do you have any thoughts on this?

Reply:
Using `quay.io/confidential-containers/runtime-payload-ci:kata-containers-latest` seems like a good idea.

Reply:
Just have a try it seems that `quay.io/confidential-containers/runtime-payload-ci:kata-containers-latest` is available now, will update the pr soon.